docs/api/framework-conventions/entry.client.tsx.md

```diff
@@ -0,0 +1,43 @@
+---
+title: entry.client.tsx
+order: 4
```

Comment:
@brophdawg11 I decided to order this:

- declarative
- data
- framework

because that's the order in [Picking a mode](https://reactrouter.com/start/modes)

This is different from what you advised on Discord, so just wanted to bring it up here